src/main/java/com/company/Main.java

```diff
@@ -12,14 +12,17 @@ public class Main {
     public static void main(String[] args) throws IOException {
```

Comment:
no takiego syfu to nie robimy. Wyjatki sie obsluguje a nie przerzuca w gore az dojdzie do maina